content/applications/finance/accounting/customer_invoices/payment_terms.rst

```diff
@@ -2,120 +2,132 @@
 Payment terms and installment plans
```

Comment:
Do we really talk about installment plans on this page? Since there's no specific section that talks about this, do we really need to have it in the title?

Let's talk about this :)